features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/DeveloperSettingsView.kt

```diff
@@ -136,22 +136,20 @@ private fun ElementCallCategory(
 ) {
     PreferenceCategory(title = "Element Call", showTopDivider = true) {
         val callUrlState = state.customElementCallBaseUrlState
-        fun isUsingDefaultUrl(value: String?): Boolean {
-            return value.isNullOrEmpty() || value == callUrlState.defaultUrl
-        }
 
-        val supportingText = if (isUsingDefaultUrl(callUrlState.baseUrl)) {
+        val supportingText = if (callUrlState.baseUrl.isNullOrEmpty()) {
             stringResource(R.string.screen_advanced_settings_element_call_base_url_description)
         } else {
             callUrlState.baseUrl
         }
         PreferenceTextField(
             headline = stringResource(R.string.screen_advanced_settings_element_call_base_url),
-            value = callUrlState.baseUrl ?: callUrlState.defaultUrl,
+            value = callUrlState.baseUrl,
+            placeholder = "https://.../room",
```

Comment:
Why a `/room` in the placeholder?

Reply:
The new EC URL path for in-app calls (room calls) uses the `/room` suffix.

Reply:
Not sure to understand, as it is the url of the service hosted somewhere (no?). So I do not see the constraint on the URL path. But OK.

Reply:
@bmarty you are correct that the URL path isn't constrained. So, I would recommend not having the placeholder value at all. This is consistent with the [iOS implementation](https://github.com/element-hq/element-x-ios/pull/3939).

Reply:
OK, thanks. Tracked by https://github.com/element-hq/element-x-android/issues/4475